packages/components/src/components/tag/tag.stories.ts

```diff
@@ -29,173 +32,90 @@ export const Default = {
 };
 
 /**
- * The tag in all possible combinations of `selected` and `size`.
+ * Use the `size` attribute to change the tag size. The default size is `lg`.
  */
 
-export const selectedAndSize = {
-  name: 'Selected × Size',
-  parameters: { controls: { exclude: ['size', 'selected'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'selected', values: ['false', 'true'] },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+export const Size = {
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-tag size="lg">Large</sd-tag>
+      <sd-tag size="sm">Small</sd-tag>
+    </div>
+  `
 };
 
 /**
- * The tag in all possible combinations of `removable` and `size`.
+ * Use the `selected` attribute to enable the selected state.
  */
 
-export const removableAndSize = {
-  name: 'Removable × Size',
-  parameters: { controls: { exclude: ['size', 'removable'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'removable', values: ['false', 'true'] },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+export const Selected = {
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-tag selected>Selected</sd-tag>
+    </div>
+  `
 };
 
 /**
- * Use the `disabled` attribute to disable a tag. Clicks will be suppressed until the disabled state is removed.
- *
- * __Hint:__ If the href attribute is set i.e. the tag is rendered as a link (`<a>`),
- * the disabled attribute is ignored, as links may not be disabled.
- * To disable the tag in this case the href attribute has to be removed as well.
+ * Use the `removable` attribute to include the removability indicator and make the tag removable from the UI.
  */
 
-export const Disabled = {
-  name: 'Disabled',
-  parameters: { controls: { exclude: ['selected', 'removable', 'disabled'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'selected', values: ['false', 'true'] },
-        y: { type: 'attribute', name: 'removable', values: ['false', 'true'] }
-      },
-      constants: [
-        {
-          type: 'attribute',
-          name: 'disabled',
-          value: 'true'
-        }
-      ],
-      args
-    });
-  }
+export const Removable = {
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-tag removable>Removable</sd-tag>
+    </div>
```

Comment:
```suggestion
    <div id="tags-removable" class="flex gap-12">
      <sd-tag size="sm" removable>Removable <span class="sr-only">small tag</span></sd-tag>
      <sd-tag size="large" removable>Removable <span class="sr-only">large tag</span></sd-tag>
    </div>
    
<script>
  const tag = document.querySelector('#tags-removable');

  div.addEventListener('sd-remove', event => {
    const tag = event.target;
    tag.style.opacity = '0';
    setTimeout(() => (tag.style.opacity = '1'), 2000);
  });
</script>
```

Maybe we can make this a bit more interactive to show how this could be handled.